articles/help/faq.md

```diff
@@ -27,7 +27,7 @@ These Modules are written outside of Intent Architect as plugins, and similar Mo
 
 ## How do I create my own patterns / modules?
 
-[Make your own Module](creating_a_module_from_scratch.md)
+[Make your own Module](xref:TutorialCreateModule)
```

Comment:
```suggestion
[](xref:TutorialCreateModule)
```

According to [DocFX Docs](https://dotnet.github.io/docfx/tutorial/links_and_cross_references.html#different-syntax-of-cross-reference), leaving the square brackets blank makes it resolve the link text automatically from the article heading.

To improve maintainability, we should consider leaving square brackets for links blank as often as possible.